### ISSUE TITLE
[BUG] Some PDFs cannot consumed, Parser Error

### ISSUE THREAD

**Issue by solariz** [paperless-ngx/paperless-ngx#1281]
### Description

I recently started using paperless ngx and already have a big amount of PDFs on my NAS, previously managed per hand in folder Structures, all of them are already OCR or partly original PDF (text) most of them show they cant import into paerless-ngx with an error:

**PdfError: operation for dictionary attempted on object of type null**

The Full Error is:
```
[2022-07-25 19:23:53,449] [INFO] [paperless.consumer] Consuming test-telekom.pdf
[2022-07-25 19:23:53,452] [DEBUG] [paperless.consumer] Detected mime type: application/pdf
[2022-07-25 19:23:53,454] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser
[2022-07-25 19:23:53,463] [DEBUG] [paperless.consumer] Parsing test-telekom.pdf...
[2022-07-25 19:23:54,074] [DEBUG] [paperless.parsing.tesseract] Extracted text from PDF file /tmp/paperless/paperless-upload-f8ybkxlg
[2022-07-25 19:23:54,180] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': '/tmp/paperless/paperless-upload-f8ybkxlg', 'output_file': '/tmp/paperless/paperless-rv5k7xma/archive.pdf', 'use_threads': True, 'jobs': 1, 'language': 'deu+eng', 'output_type': 'pdfa', 'progress_bar': False, 'skip_text': True, 'clean': True, 'deskew': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': '/tmp/paperless/paperless-rv5k7xma/sidecar.txt'}
[2022-07-25 19:23:54,844] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-rv5k7xma
[2022-07-25 19:23:54,850] [ERROR] [paperless.consumer] Error while consuming document test-telekom.pdf: PdfError: operation for dictionary attempted on object of type null: returning null for attempted key retrieval
Traceback (most recent call last):
  File "/usr/src/paperless/src/paperless_tesseract/parsers.py", line 261, in parse
    ocrmypdf.ocr(**args)
  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/api.py", line 337, in ocr
    return run_pipeline(options=options, plugin_manager=plugin_manager, api=True)
  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_sync.py", line 385, in run_pipeline
    exec_concurrent(context, executor)
  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_sync.py", line 297, in exec_concurrent
    pdf = ocrgraft.finalize()
  File "/usr/local/lib/python3.9/site-packages/ocrmypdf/_graft.py", line 198, in finalize
    self.pdf_base.save(self.output_file)
  File "/usr/local/lib/python3.9/site-packages/pikepdf/_methods.py", line 774, in save
    self._save(
pikepdf._qpdf.PdfError: operation for dictionary attempted on object of type null: returning null for attempted key retrieval
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/consumer.py", line 261, in try_consume_file
    document_parser.parse(self.path, mime_type, self.filename)
  File "/usr/src/paperless/src/paperless_tesseract/parsers.py", line 314, in parse
    raise ParseError(f"{e.__class__.__name__}: {str(e)}")
documents.parsers.ParseError: PdfError: operation for dictionary attempted on object of type null: returning null for attempted key retrieval
```

I attached one file where this happens and removed the pages containing private data. Still happening to me.

[test-telekom.pdf](https://github.com/paperless-ngx/paperless-ngx/files/9183546/test-telekom.pdf)

### Steps to reproduce

1. Upload one or many PDF
2. Processing start with the described outcome on top
3. Process halts with error

### Paperless-ngx version

1.7.1

### Host OS

Ubuntu 20.04 x86_64

### Installation method

Docker - official image

**Comment by shamoon** [comment]
This sounds like a duplicate of #868 , I believe the issue is a bug in one of our dependencies and is fixed in our upcoming 1.8 release

**Comment by solariz** [comment]
Thanks, is there a way to push the docker image to the RC branch? Or when is 1.8 planned for release?
Converting hundrets of PDF manually before import is something I would like to avoid ;)

Thanks

**Comment by stumpylog** [comment]
You could update to the `:beta` tag, which should be pretty stable at this point.  Hopefully it will be fully released soon, but there's not an exact timeline for it.

**Comment by stumpylog** [comment]
With 1.8.0 released, pikepdf is now at 5.4.0, so I'm closing this as it should be resolved with the new version.
